release_notes.md

```diff
@@ -0,0 +1,38 @@
+ # Release notes
+
+- Woof Mobile APP
+
+All notable changes to this application will be documented in this file.
+
+## [Unreleased]
+
+`More information about working with the application will be added during development.`
+
+### User impacts
+
+Software requirements: 
+
+ * Deployment target for iOS SDKs - 15.0+
+ * Supported devices - iPhone only
+ * Supported modes - portrait
+ * Dark/night theme - deactivate
+ * The linters (Swiftformat and Swiftlint) integrated with Xcode in the build phase.
+
+### Notes
+
+Anyone can download and use the app features.
+
+### Support
+
+You can contact the authors by the following GitHub's usernames:
+
+- Nadia Shpakouskaya: [@NadzeyaShpakouskaya](https://github.com/NadzeyaShpakouskaya)
+- Alex Zavadskiy: [@iBYarchitect](https://github.com/iBYarchitect)
+- Julya Kmita: [@julyakmita](https://github.com/julyakmita)
+
+### Added 
+- Users can now download the initial Xcode project.
+
+## [0.1.0]
```

Comment:
Why do we have both `unreleased` and `0.1.0` section.
I don't understand where to put the info when I need to update the file.

Reply:
We decided  with the team to to delete this information. In previous review you wrote that you also don't mind. Now the file release_notes.md keeps only the title of the file and the app name, because we haven't made any releases yet.